products/react.md

```diff
@@ -26,34 +26,13 @@ releases:
     latest: "19.0.0"
     latestReleaseDate: 2024-12-05
 
--   releaseCycle: "18.3"
-    releaseDate: 2024-04-25
-    eoas: false
+-   releaseCycle: "18"
+    releaseDate: 2022-03-29
+    eoas: 2024-04-26
```

Comment:
the end of active support for react 18 can’t have occurred before react 19 came out?

Reply:
yes, I don't know if this  branch is eoas, so I can set this value to false what do you think ?

Reply:
according to https://github.com/reactjs/react.dev/issues/1745#issuecomment-466767389 when a new major version is releases older one are EOL.
So I recommend to review all releases before 19.X to flag them with : 
eol: true